### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by FrauR**
update README.md according to the changes and new  features.

### CODE REVIEW

**Comment by FrauR** [comment]
@michaelengland could you please extend in case I forgot anything?

File: README.md [hunk 1]
```diff
@@ -19,24 +19,26 @@ How can we get the benefits of working on `master` branch, while still not worry
 
 ## Introducing FlipTheSwitch
 
-With FlipTheSwitch, we can choose different code paths at runtime:
+### Auto-Generated Files:
+
+With 'FTSFlipTheSwitch', we can choose different code paths at runtime:
```
Comment (michaelengland) [inline README.md]: This should go back above the title, and just be `With FlipTheSwitch, we can...` again.

**Comment by michaelengland** [comment]
This seems to have a load of extra commits in it, are they old?

**Comment by michaelengland** [comment]
I think we also need additional functionality sub-headers for #22 & #21 , as well as adding these to the CONTRIBUTING.md

**Comment by FrauR** [comment]
@michaelengland can you check please?

**Comment by michaelengland** [comment]
:+1:

**Pull request merged**